apps/browser/qml/pages/components/PopUpMenuItem.qml

```diff
@@ -0,0 +1,222 @@
+            width: parent.width
+
+            OverlayListItem {
+                height: Theme.itemSizeSmall
+                iconWidth: root.iconWidth
+                horizontalOffset: root.horizontalOffset
+                iconSource: "image://theme/icon-m-search-on-page"
+                enabled: webView.contentItem
+                //% "Search on page"
+                text: qsTrId("sailfish_browser-la-search_on_page")
+
+                onClicked: {
+                    overlay.toolBar.findInPageActive = true
+                    overlay.toolBar.findInPage()
+                }
+            }
+
+            OverlayListItem {
+                height: Theme.itemSizeSmall
+                iconWidth: root.iconWidth
+                horizontalOffset: root.horizontalOffset
+                iconSource: "image://theme/icon-m-add-to-grid"
+                //% "Add to apps grid"
+                text: qsTrId("sailfish_browser-la-add_to_apps_grid")
+                onClicked: {
+                    overlay.animator.showChrome()
+                    pageStack.animatorPush("AddToAppGridDialog.qml",
+                                                  {
+                                                      "url": url,
+                                                      "title": title,
+                                                      "desktopBookmarkWriter": desktopBookmarkWriter,
+                                                      "bookmarkWriterParent": pageStack
+                                                  })
+                }
+            }
+
+            OverlayListItem {
+                height: Theme.itemSizeSmall
+                iconWidth: root.iconWidth
+                horizontalOffset: root.horizontalOffset
+                iconSource: "image://theme/icon-m-file-download-as-pdf"
+                //% "Save web page as PDF"
+                text: qsTrId("sailfish_browser-la-save_as-pdf")
+
+                onClicked: {
+                    overlay.animator.showChrome()
+                    if (DownloadManager.pdfPrinting) {
+                        pdfPrintingNotice.show()
+                    } else {
+                        overlay.toolBar.savePageAsPDF()
+                    }
+                }
+            }
+        }
+
+        OverlayListItem {
+            enabled: webView.contentItem
+            height: Theme.itemSizeSmall
+            iconWidth: root.iconWidth
+            horizontalOffset: root.horizontalOffset
+            checkable: true
+            checked: webView.contentItem && webView.contentItem.desktopMode
+            iconSource: "image://theme/icon-m-computer"
+            //: Label for text switch that reloads page in desktop mode
+            //% "Desktop version"
+            text: qsTrId("settings_browser-la-desktop_version")
+
+            onClicked: {
+                overlay.animator.showChrome()
+                webView.contentItem.desktopMode = !webView.contentItem.desktopMode
+            }
```

Comment:
The Desktop version button looks nice now, thanks, especially now it has the highlight bar behind it when pressed like the other items in the menu.